Python/ceval.c

```diff
@@ -1481,9 +1480,9 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
     f->f_stackdepth = -1;
     f->f_state = FRAME_EXECUTING;
 
-    if (co->co_opcache_flag < OPCACHE_MIN_RUNS) {
+    if (co->co_opcache_flag < opcache_min_runs) {
         co->co_opcache_flag++;
-        if (co->co_opcache_flag == OPCACHE_MIN_RUNS) {
+        if (co->co_opcache_flag == opcache_min_runs) {
```

Comment:
This may increase memory access in hot path. Would you run pypeformance to ensure no performance regression?

Reply:
Sure, will run this over the weekend.

Reply:
```
❯ pyperf compare_to json/* --table --min-speed 2 -G
+----------------+--------------------------------------+-----------------------------------------+
| Benchmark      | 2021-02-27_11-31-master-c71d24f55828 | 2021-02-27_19-33-bpo-37136-a9680bcf9b84 |
+================+======================================+=========================================+
| unpickle       | 17.0 us                              | 17.4 us: 1.02x slower                   |
+----------------+--------------------------------------+-----------------------------------------+
| pickle         | 13.2 us                              | 13.7 us: 1.04x slower                   |
+----------------+--------------------------------------+-----------------------------------------+
| nqueens        | 121 ms                               | 126 ms: 1.04x slower                    |
+----------------+--------------------------------------+-----------------------------------------+
| Geometric mean | (ref)                                | 1.00x slower                            |
+----------------+--------------------------------------+-----------------------------------------+

Benchmark hidden because not significant (56): xml_etree_parse, scimark_sor, unpickle_list, nbody, unpack_sequence, regex_v8, chaos, meteor_contest, telco, python_startup, genshi_text, regex_compile, python_startup_no_site, json_loads, regex_dna, logging_simple, sympy_sum, xml_etree_iterparse, scimark_sparse_mat_mult, xml_etree_process, richards, chameleon, sqlalchemy_declarative, django_template, hexiom, pathlib, json_dumps, tornado_http, sympy_str, unpickle_pure_python, pyflate, pidigits, go, regex_effbot, deltablue, sympy_expand, xml_etree_generate, float, genshi_xml, logging_silent, raytrace, fannkuch, sqlalchemy_imperative, scimark_monte_carlo, logging_format, 2to3, sympy_integrate, mako, pickle_dict, pickle_list, spectral_norm, dulwich_log, sqlite_synth, scimark_lu, pickle_pure_python, scimark_fft, crypto_pyaes
```